src/mkfs.fat.c

```diff
@@ -1234,7 +1235,7 @@ static void setup_tables(void)
 #define seekto(pos,errstr)						\
   do {									\
     off_t __pos = (pos);						\
-    if (lseek (dev, __pos, SEEK_SET) != __pos)				\
+    if (lseek (dev, part_offset + __pos, SEEK_SET) != part_offset + __pos)				\
```

Comment:
... and this looks like that `part_offset` is treated in bytes.

Reply:
Yes. part_offset is treated in bytes. See also

https://github.com/dosfstools/dosfstools/pull/84/files#diff-47b7a74c103277bcad9443e559f1ee9bR1603

Reply:
Yes. See also:

https://github.com/dosfstools/dosfstools/pull/84/files#diff-47b7a74c103277bcad9443e559f1ee9bR1603